examples/concentrating_solar_power/csp_plant_example.py

```diff
@@ -48,6 +42,13 @@
 temp_collector_inlet = 435
 temp_collector_outlet = 500
 
+# input data
+dataframe = pd.read_csv(data_path + 'data_csp_plant.csv').head(periods)
+dataframe['Datum'] = pd.to_datetime(dataframe['Datum'])
+dataframe.set_index('Datum', inplace=True)
+dataframe.index = dataframe.index.tz_localize(tz='Asia/Muscat')
```

Comment:
```suggestion
dataframe.set_index('Datum', inplace=True)
dataframe = dataframe.asfreq('H')
dataframe.index = dataframe.index.tz_localize(tz='Asia/Muscat')
```

I somehow also get the same error mentioned by @jnnr in the comment https://github.com/oemof/oemof-thermal/pull/77#issuecomment-608494618 running the script if I don't assign 'H' to freq.

I have oemof 0.3.2 and pandas 0.25.3 versions.

Reply:
Thanks, I just changed this in one of the two example files.